File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.js
function solution(numbers) {
    var num = numbers.sort((a,b)=>b-a);
    var a_b = num[num.length-1]*num[num.length-2];
    var b_a = num[0]*num[1]
    var sum = 0;
    a_b > b_a ? sum = a_b : sum = b_a
    return sum;
}

File: 프로그래머스/0/120862. 최댓값 만들기 （2）/최댓값 만들기 （2）.kt
class Solution {
    fun solution(numbers: IntArray): Int {
        var answer: Int = 0
        numbers.sort()
        var numOfNumbers = numbers.size
        var numbersFirst = numbers[0]*numbers[1]
        var numbersLast = numbers[numOfNumbers-1]*numbers[numOfNumbers-2]
        
        answer = if(numbersFirst > numbersLast) 
            numbersFirst
        else if(numbersFirst < numbersLast)
            numbersLast
        else
            numbersLast
            
        return answer
    }
}